lab-carolina/karma.conf.js

```diff
@@ -0,0 +1,75 @@
+// Karma configuration
```

Comment:
Go ahead and remove all these comments from the config file after it's generated. It'll help clean things up.